tools/monorepo-utils/src/slack/commands/slack/slack-message.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../../../core/logger';
+import { requestAsync } from '../../../core/util';
+
+type SlackResponse = {
```

Comment:
Greatly simplified version of the slack response, but its all we need for what we're handling in this command.